src/ebay-calendar/date-utils.ts

```diff
@@ -0,0 +1,48 @@
+export type DayISO = `${number}-${number}-${number}`;
+
+export function findFirstDayOfWeek(localeName: string): number {
+    // weekInfo only exists on some browsers, so we default to Sunday otherwise
+    // https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/Locale/weekInfo
+
+    const locale = new Intl.Locale(localeName) as Intl.Locale & {
+        weekInfo?: {firstDay: number};
+    }
+    if (locale.weekInfo) {
+        return locale.weekInfo.firstDay
+    }
+    return 0
+}
+
+export function getWeekdayInfo(localeName: string) {
```

Comment:
@abiyasa @darkwebdev I usually prefer to let TypeScript define the returned type, do we still need this eslint rule?

Reply:
@HenriqueLimas While it make sense to have implicit return type for trivia or obvious functions, but it's more beneficial to have explicit return type, especially for maintainability.

With signature `function getWeekdayInfo(localeName: string): DayISO`:

- Engineers can quickly understand the function contract (input and output type) without having to check the implementation.
- If someone later change the function implementation to also return null or number, TypeScript will give compilation error. This will prevent accidentally change and could help the code review: easier to tell if the type change is intentionally or accidental.

Reply:
That make sense, thank you let me fix that. Maybe we should move as error then instead of warning